test/cluster_helper.go

```diff
@@ -47,6 +47,131 @@ var (
 	pre6Image          = fmt.Sprintf("%s:%s", baseImage, pre6Version)
 )
 
+func rollingRestartClusterByEnablingTLS(
+	k8sClient client.Client, ctx goctx.Context,
+	clusterNamespacedName types.NamespacedName,
+) error {
+	aeroCluster, err := getCluster(k8sClient, ctx, clusterNamespacedName)
+	if err != nil {
+		return err
+	}
+
+	// Change tls conf
+	aeroCluster.Spec.OperatorClientCertSpec = &asdbv1.AerospikeOperatorClientCertSpec{
+		AerospikeOperatorCertSource: asdbv1.AerospikeOperatorCertSource{
+			SecretCertSource: &asdbv1.AerospikeSecretCertSource{
+				SecretName:         tlsSecretName,
+				CaCertsFilename:    "cacert.pem",
+				ClientCertFilename: "svc_cluster_chain.pem",
+				ClientKeyFilename:  "svc_key.pem",
+			},
+		},
+	}
+	aeroCluster.Spec.AerospikeConfig.Value["network"] = getNetworkTLSConfig()
+
+	err = k8sClient.Update(ctx, aeroCluster)
+	if err != nil {
+		return err
+	}
+
+	err = waitForAerospikeCluster(
+		k8sClient, ctx, aeroCluster, int(aeroCluster.Spec.Size), retryInterval,
+		getTimeout(aeroCluster.Spec.Size),
+	)
```

Comment:
Can use `updateCluster( k8sClient, ctx, aeroCluster)` instead.
Do this wherever applicable in test-cases